Supertext.Base.Dal.SqlServer/Utils/SqlResultConverter.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+
+namespace Supertext.Base.Dal.SqlServer.Utils
+{
+    public class SqlResultConverter : ISqlResultConverter
+    {
+        public void InterpretUtcDates(IEnumerable<dynamic> results)
```

Comment:
Can't the arguments be of type `IEnumerable<IDictionary<string, object>>`? Then `dynamic` wouldn't be used.